ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -31,12 +31,53 @@
 
 
 def _get_fn_dtypes(framework, kind="valid"):
+    """
+    Return a list of supported data types for a specified machine learning framework and kind of data type.
+
+    Parameters
+    ----------
+    framework
+        A machine learning framework object.
+
+    kind
+        the kind of data type being queried. The default value is 'valid',
+        which returns al the valid data types for the framework.
+
+    Returns
+    -------
+        A list of supported data types for the specified framework and kind of data type.
+    """
     return test_globals.CURRENT_RUNNING_TEST.supported_device_dtypes[framework.backend][
         test_globals.CURRENT_DEVICE_STRIPPED
     ][kind]
 
 
 def _get_type_dict(framework, kind):
+    """
+
+    Returns a dictionary of valid data types for a specified machine learning framework and kind of data type.
+
+    Parameters
+    ----------
+    framework
+        A machine learning framework object.
+    kind
+        The kind of data type being queried. Possible values are:
+            -"valid": all valid data types for the framework
+            -"numeric": all valid numeric data types for the framework
+            -"integer": all valid integer data types for the framework
+            -"float": all valid floating-point data types for the framework
+            -"unsigned": all valid unsigned integer data types for the framework
+            -"signed_integer": all valid signed integer data types for the framework
+            -"complex": all valid complex data types for the framework
+            -"real_and_complex": all valid numeric and complex data types for the framework
+            -"float_and_complex": all valid floating-point and complex data types for the framework
+            -"bool": all valid boolean data types for the framework
+
+    Returns
+    -------
+        dict: A dictionary of valid data types for the specified framework and kind of data type.
```

Comment:
Let's make a clear distinguish here that `valid` here is different from the value for the parameter `kind`

Reply:
I changed it to "A dictionary of all accepted data type for the specified framework ad kind of data type", Does that sound okay?